common/logisticspipes/utils/gui/SimpleGraphics.java

```diff
@@ -262,10 +259,18 @@ public static void drawQuad(Tessellator tessellator, int x, int y, int width, in
 	public static void drawQuad(Tessellator tessellator, int x, int y, int width, int height, int color, double zLevel) {
 		BufferBuilder buf = tessellator.getBuffer();
 		buf.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION_COLOR);
-		buf.pos(x, y, zLevel).color(Color.getRed(color), Color.getGreen(color), Color.getBlue(color), Color.getAlpha(color)).endVertex();
-		buf.pos(x, y + height, zLevel).color(Color.getRed(color), Color.getGreen(color), Color.getBlue(color), Color.getAlpha(color)).endVertex();
-		buf.pos(x + width, y + height, zLevel).color(Color.getRed(color), Color.getGreen(color), Color.getBlue(color), Color.getAlpha(color)).endVertex();
-		buf.pos(x + width, y, zLevel).color(Color.getRed(color), Color.getGreen(color), Color.getBlue(color), Color.getAlpha(color)).endVertex();
+		float a = Color.getAlpha(color);
+		float r = Color.getRed(color);
+		float g = Color.getGreen(color);
+		float b = Color.getBlue(color);
+		buf.pos(x, y, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x, y + height, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x + width, y + height, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x + width, y, zLevel)
+				.color(r, g, b, a).endVertex();
```

Comment:
adding newlines before `.color` doesn't seem necessary and looks out of place

Reply:
I like it better since the pos() has variable length, but if you want it on one line I will change it.

Reply:
I am fine with it, if you put `.endVertex();` on a new line as well, since that makes is less random while improving readability